kubemarine/core/utils.py

```diff
@@ -295,6 +295,21 @@ def load_yaml(filepath) -> dict:
         do_fail(f"Failed to load {filepath}", exc)
 
 
+def true_or_false(value):
```

Comment:
I would also suggest to consider:
* SystemdCgroup in containerd.configure
* enabled in coredns.generate_nested_sections

Reply:
In this PR I fix only plugins.install. 
Of course this problem appears for other values, but we should design a generic approach to fix all places.
Ticket number for it: MANOPD-82348